lib/ddtrace/utils/rails.rb

```diff
@@ -0,0 +1,12 @@
+module Datadog
+  module Utils
+    # Common rails-related utility functions.
+    module Rails
+      module_function
+
+      def railtie_supported?
```

Comment:
Because this file relates to an integration (Rails), it should be moved to the `contrib` directory. I noticed we already have a Rails `Util` module, so I suggest moving this method there: https://github.com/DataDog/dd-trace-rb/blob/v0.43.0/lib/ddtrace/contrib/rails/utils.rb

Reply:
yup agreed, updated